xml/snapper.xml

```diff
@@ -1889,7 +1909,7 @@ local  | /local</screen>
       <listitem>
        <para>
         Snapshot of a file system <emphasis>before</emphasis> a modification.
-        Each <literal>pre</literal> snapshot has got a corresponding
+        Each <literal>pre</literal> snapshot corresponds to a
         <literal>post</literal> snapshot. Used for the automatic &yast;/Zypper
```

Comment:
you can make multiline suggestions (which would make this situation much less confusing):

Select multiple lines by dragging on their line numbers, then add a `\`\`\`suggestion` and one or more lines to the comment. :)

Otherwise, agreed.

Reply:
Aha! I did not know that. Thanks.